CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [1.5.0] 2022-11-08
```

Comment:
keep everything under the 1.4.0 version (that's the components release version, we don't need multiple ones)